core/sts-3.3.md

```diff
@@ -205,7 +241,8 @@ discussed in General security considerations.
 As further protection against bootstrap MITM vulnerabilities, clients may choose to
 include a pre-loaded list of known hosts with STS policies. Such lists should be
 compiled on an opt-in basis, by request of IRC network administrators. Hosts should be
-verified to be correctly advertising an STS policy before inclusion.
+verified to be correctly advertising an STS policy, as well as a 
+[preload policy](#the-preload-key) before inclusion.
```

Comment:
Could you make it explicit that `preload` implies “by request of IRC network administrators”?